MAUI/TreeMap/getting-started.md

```diff
@@ -0,0 +1,571 @@
+    }
+}
+
+{% endhighlight %}
+
+{% highlight c# tabtitle="PopulationViewModel.cs" %}
+
+public class PopulationViewModel
+{
+    public PopulationViewModel()
+    {
+        this.PopulationDetails = new ObservableCollection<PopulationDetails>()
+            {
+                new PopulationDetails() { Continent ="North America", Country = "United States of America", Population = 339996564 },
+                new PopulationDetails() { Continent ="South America", Country = "Brazil", Population = 216422446 },
+                new PopulationDetails() { Continent ="North America", Country = "Mexico", Population = 128455567 },
+                new PopulationDetails() { Continent ="South America", Country = "Colombia", Population = 52085168 },
+                new PopulationDetails() { Continent ="South America", Country = "Argentina", Population = 45773884 },
+                new PopulationDetails() { Continent ="North America", Country = "Canada", Population = 38781292 },
+                new PopulationDetails() { Continent ="South America", Country = "Peru", Population = 34352719 },
+                new PopulationDetails() { Continent ="South America", Country = "Venezuela", Population = 28838499 },
+                new PopulationDetails() { Continent ="South America", Country = "Chile", Population = 19629590 },
+                new PopulationDetails() { Continent ="South America", Country = "Ecuador", Population = 18190484 },
+                new PopulationDetails() { Continent ="North America", Country = "Guatemala", Population = 18092026 },
+                new PopulationDetails() { Continent ="South America", Country = "Bolivia", Population = 12388571 },
+                new PopulationDetails() { Continent ="North America", Country = "Honduras", Population = 10593798 },
+                new PopulationDetails() { Continent ="North America", Country = "Nicaragua", Population = 7046311 },
+                new PopulationDetails() { Continent ="South America", Country = "Paraguay", Population = 6861524 },
+                new PopulationDetails() { Continent ="North America", Country = "El Salvador", Population = 6364943 },
+                new PopulationDetails() { Continent ="North America", Country = "Costa Rica", Population = 5212173 },
+                new PopulationDetails() { Continent ="South America", Country = "Uruguay", Population = 3423109 },
+            };
+    }
+    public ObservableCollection<PopulationDetails> PopulationDetails
+    {
+        get;
+        set;
+    }
+}
+
+{% endhighlight %}
+{% highlight c# tabtitle="PopulationDetails.cs" %}
+
+public class PopulationDetails
+{
+    public string Country { get; set; }
+    public string Continent { get; set; }
+    public int Population { get; set; }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+
+## Enable tooltip
+
+You can use `ShowToolTip` property of SfTreeMap to enable tooltip for the treemap control. The default value of `ShowToolTip` is `false`.
```

Comment:
To enable tooltips for the TreeMap control, utilize the ShowToolTip property of SfTreeMap. By default, the value of ShowToolTip is set to false. To provide users with additional information or context about specific treemap items, simply set this property to true.

Reply:
Content changed